controllers/admin/AdminBackupController.php

```diff
@@ -318,4 +318,83 @@ public function strSort($a, $b)
         return $this->_orderWay == 'ASC' ? strcmp($a[$this->sort_by], $b[$this->sort_by]) :
             strcmp($b[$this->sort_by], $a[$this->sort_by]);
     }
+
+    /**
+     * Reads and output the content of a backup file
+     *
+     * Expected query parameter {filename}
+     */
+    public function displayAjaxBackupContent()
```

Comment:
You need to rebase, we have migrated this controller to Symfony

Reply:
hmm I will probably need to revert a commit if we want to avoid a BC break.